dtfs-central-api/src/v1/controllers/tfm/facility/tfm-get-facilities.controller.js

```diff
@@ -86,7 +86,7 @@ exports.getAllFacilities = async (req, res) => {
                     },
                     {
                       case: { $eq: ['$dealSnapshot.dealType', 'BSS/EWCS'] },
-                      then: { $concat: ['$$facilities.coverEndDate-month', '-', '$$facilities.coverEndDate-day', '-', '$$facilities.coverEndDate-year'] },
+                      then: { $concat: ['$$facilities.coverEndDate-year', '-', '$$facilities.coverEndDate-month', '-', '$$facilities.coverEndDate-day'] },
```

Comment:
GEF had the following format: `2021-10-08` and for BSS I had to replicate the same format; hence this change ^